interp/cfg.go

```diff
@@ -740,7 +740,18 @@ func (interp *Interpreter) cfg(root *node, importPath string) ([]*node, error) {
 			wireChild(n)
 			t := n.child[0].typ
 			switch t.cat {
-			case aliasT, ptrT:
+			case aliasT:
```

Comment:
instead of duplicating the code, we could keep them together, and only do the first check if we are an alias, no?

```
case aliasT, ptrT:
   if t.cat == aliasT && isString(t.val.TypeOf()) {
...
```